microreactors/gcmr/assembly/RIA_steady_state/Griffin_steady_state.i

```diff
@@ -0,0 +1,227 @@
+    type = GeneratedMeshGenerator
+    dim = 3
+    nx = 10
+    ny = 10
+    nz = 20
+    xmin = -0.1
+    xmax = 0.1
+    ymin = -0.1
+    ymax = 0.1
+    zmin = 0.
+    zmax = 2.
+  []
+  [assign_coarse_id]
+    type = CoarseMeshExtraElementIDGenerator
+    input = fmg_id
+    coarse_mesh = coarse_mesh
+    extra_element_id_name = coarse_element_id
+  []
+[]
+
+[Executioner]
+  type = SweepUpdate
+
+  richardson_abs_tol = 1e-8
+  richardson_rel_tol = 1e-8
+  richardson_value = eigenvalue
+
+  richardson_max_its = 1000
+  inner_solve_type = GMRes
+  max_inner_its = 20
+
+  fixed_point_max_its = 1
+  custom_pp = eigenvalue
+  custom_rel_tol = 1e-6
+  force_fixed_point_solve = true
+
+  cmfd_acceleration = true
+  coarse_element_id = coarse_element_id
+  cmfd_eigen_solver_type = newton
+  prolongation_type = multiplicative
+  max_diffusion_coefficient = 1
+[]
+
+[Debug]
+  check_boundary_coverage = false
+  print_block_volume = false
+  show_actions = false
+[]
+
+[AuxVariables]
+  [Tf]
+    initial_condition = 873.15
+    order = CONSTANT
+    family = MONOMIAL
+  []
+[]
+
+[TransportSystems]
+  particle = neutron
+  equation_type = eigenvalue
+
+  G = 11
+  VacuumBoundary = '2000 2001'
+  ReflectingBoundary = '9000'
+
+  [SN]
+    scheme = DFEM-SN
+    family = MONOMIAL
+    order = FIRST
+
+    AQtype = Gauss-Chebyshev
+    NPolar = 2
+    NAzmthl = 3
+    NA = 2
+    n_delay_groups = 6
+
+    using_array_variable = true
+    collapse_scattering = true
+    hide_angular_flux = true
+  []
+[]
+
+[PowerDensity]
+  power = 225e3 # Assembly Power from NS
+  power_density_variable = power_density
+  integrated_power_postprocessor = integrated_power
+[]
+
+[Materials]
+  [mod]
+    type = CoupledFeedbackMatIDNeutronicsMaterial
+    block = '10 100 102 103 200 201 400 401 500 600 8000 8001'
+    library_file = '../ISOXML/XS_Griffin.xml'
+    library_name = XS_Griffin
+    isotopes = 'pseudo'
+    densities = 1.0
+    is_meter = true
+    # power normalization
+    plus = true
+    dbgmat = false
+    grid_names = 'Tmod'
+    grid_variables = 'Tf'
+  []
+  [Air]
+   type = CoupledFeedbackRoddedNeutronicsMaterial
+   block = '300'
+   library_file = '../ISOXML/XS_Griffin.xml'
+   library_name = XS_Griffin
+   rod_segment_length = '1.2 0.4'
+   rod_withdrawn_direction = z
+   isotopes = 'pseudo; pseudo; pseudo; pseudo'
+   densities = '1.0 1.0 1.0 1.0'
+   segment_material_ids = '804 809 809 805'
+   front_position_function = control_rod_position
+   diffusion_coefficient_scheme = user_supplied
+   is_meter = true
+   plus = true
+   dbgmat = false
+   grid_names = 'Tmod'
+   grid_variables = 'Tf'
+  []
+[]
+
+[Functions]
+  [control_rod_position]
+    type = ParsedFunction
+    value = '1.4'
+  []
+[]
+
+[MultiApps]
+  [bison]
+    type = FullSolveMultiApp
+    input_files = BISON.i
+    execute_on = 'timestep_end'
+    keep_solution_during_restore = true
+  []
+[]
+
+[Transfers]
+  [to_sub_power_density]
+    type = MultiAppProjectionTransfer
```

Comment:
The two meshes should be the same in this case (the BISON mesh may have some subdomains removed). So this approach should be okay.

Reply:
are the variables the same finite element families?

this approach does not support distributed meshes I think